connector/datadogconnector/connector.go

```diff
@@ -128,7 +169,27 @@ func (c *traceToMetricConnector) run() {
 			}
 			var mx pmetric.Metrics
 			var err error
+			// Enrich the stats with container tags
+			if len(c.resourceAttrs) > 0 {
+				for _, stat := range stats.Stats {
+					if stat.ContainerID != "" {
+						if tags, ok := c.containerTagCache.Get(stat.ContainerID); ok {
+							tagList := tags.(map[string]struct{})
+							// Add unique tags to the stats
+							for _, tag := range stat.Tags {
```

Comment:
Why is this needed? As I understand `stat.Tags` is always empty in concentrator?

Reply:
if it empty - the loop would be noop.